src/Roundcube/ImapClient.php

```diff
@@ -440,6 +440,9 @@ protected function eof()
      */
     protected function closeSocket()
     {
+        if ($this->fp === null) {
+            return;
+        }
         @fclose($this->fp);
```

Comment:
Das @ kann hier auch entfernt werden. Mit PHP 8.0 unterdrückt das @ keine Fehler mehr, wie wir so unschön gemerkt haben. Daher sehe ich auch keinen Sinn das @ hier zu behalten.
```suggestion
        fclose($this->fp);
```